ui/css/treeherder-details-panel.css

```diff
@@ -134,6 +134,12 @@ div#details-panel-content .actionbar-nav > li > button:focus {
 
 .react-tabs__tab-panel--selected {
   height: 100%;
+  margin: 0.5em 1em;
+  overflow-y: auto;
+}
+
+.react-tabs__tab--disabled {
```

Comment:
There is a property `disabledTabClassName` that might work instead of doing this custom.

https://www.npmjs.com/package/react-tabs#disabledtabclassname-string

Reply:
Also for selected: `selectedTabClassName`

I know the `react-tabs__tab-panel--selected` class was already there.  Up to you if it's easier to use these props.

Reply:
If I read the documentation correctly, I'm actually not doing it custom since the default of that property is `react-tabs__tab--disabled`. Using the property `disabledTabClassName` is for defining something custom.

Reply:
For instance `react-tabs__tab--selected` is the default value and is in use when inspecting the element.
<img width="610" alt="image" src="https://user-images.githubusercontent.com/44410/88074206-1a247400-cb45-11ea-9e93-c6f82f2179ee.png">

I hope I'm not misunderstanding!

Reply:
I think this is fine.  I believe the difference is that having our CSS do this means it will apply to all `react-tabs` that we have in TH.  We only have 1, so that's fine.  :) If we had multiple and wanted them to be different, using a custom class with the props they provide would allow us to do that.